src/context.rs

```diff
@@ -462,20 +463,30 @@ impl Default for Context<'_> {
     }
 }
 
+/// Checks if a config string contains multiple files
 fn home_dir(env: &Env) -> Option<PathBuf> {
-    if cfg!(test) {
-        if let Some(home) = env.get_env("HOME") {
-            return Some(PathBuf::from(home));
-        }
+    // Try to get HOME from environment first (mocked or real)
+    if let Some(home) = env.get_env("HOME") {
+        return Some(PathBuf::from(home));
     }
-    utils::home_dir()
+    // If HOME is not set, fallback to system home directory
+    crate::utils::home_dir()
 }
 
 fn get_config_path_os(env: &Env) -> Option<OsString> {
-    if let Some(config_path) = env.get_env_os("STARSHIP_CONFIG") {
-        return Some(config_path);
+    if let Some(config_line) = env.get_env_os("STARSHIP_CONFIG") {
+        let config_str = config_line.to_str()?;
+        if config_str.is_empty() {
+            let default_path = home_dir(env)?.join(".config").join("starship.toml");
+            return Some(default_path.into_os_string());
+        }
+        // Devuelve siempre el valor de STARSHIP_CONFIG, sea uno o varios paths
```

Comment:
Could you please update this comment to English for consistency with the rest of the codebase?